clients/admin-ui/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "fidesui-template",
+  "name": "fidesops-admin-ui",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "dev:mock": "echo '🚨 Running with mock API'; NEXT_PUBLIC_MOCK_API=true next dev",
-    "build": "next build",
+    "build": "next build && next export",
```

Comment:
we'll probably want to standardize this across the repos. fidesctl looks like this right now: https://github.com/ethyca/fides/blob/aking-706-edit-collection/clients/admin-ui/package.json/#L15-L17

Reply:
Yeah. Do you know off the top of your head if Next can export to a location in a parent directory? Would simplify a lot of the copy / paste / file moving stuff.

Reply:
looks like the answer is no, it has to stay in the project dir 

https://nextjs.org/docs/api-reference/next.config.js/setting-a-custom-build-directory

> distDir should not leave your project directory. For example, ../build is an invalid directory.